pages/docs/manual/latest/record.mdx

```diff
@@ -20,7 +20,23 @@ A record needs a mandatory type declaration:
 ```res prelude
 type person = {
   age: int,
-  name: string
+  name: string,
+}
+```
+```js
+// Empty output
+```
+
+</CodeTab>
+
+The value of the record fields can be declared as `option`, just like you'd expect. In addition, the presence of the record fields themselves can also be specified as optional, by putting a `?` after the field name:
```

Comment:
I would drop the mention of `option`, or explain the difference.
Also, it should say upfront that this is from version `10.0`. I see that mentioned below, but I guess it should be mentioned every time? Definitely the first time.

Reply:
I guess 2 possilble patterns are:

1 Optional Fields (since rescript 10.0) ...
for every piece of doc

2 move all the optional fields paragraphs in one place

which one I guess depends on which learning journey one wants to optimize
someone who learns about records for the first time probably gets a gentler introduction with the latter choice
someone who's looking for updates specifically, might benefit from having all the update docs in one place